packages/v4/src/content/design-guidelines/usage-and-behavior/usage-and-behavior.md

```diff
@@ -0,0 +1,119 @@
+---
+id: Usage and behavior
+section: guidelines
+---
+
+## Displaying structured data
+Structured data includes any information that is stored in a database or similar regular data structure. The most common ways of presenting structured data are to display it in either a list or a table where rows correspond to records in the database. PatternFly supports two controls for displaying this type of data: the [Data list](/components/data-list) and the [Table](/components/table).
+
+Tables are built from a standard tabular structure of rows and columns. Data lists support any valid HTML layout inside of a row and therefore allow more formatting flexibility.
+
+|                         | [Table](/components/table) | [Data list](/components/data-list) |
```

Comment:
Feels like the left column should have a header.. like "Use case" or "Scenario"

Reply:
@afialkoff had a similar comment when she reviewed the Google doc version. I did not feel a title here would add much, but can you take a look at this again now that it's viewable in the website preview @afialkoff ?  If you both think it will add value then I'm good to add a heading above the first column.

Reply:
Hm... After looking at the preview, my gut says adding that heading would be valuable — if only to balance out the top row. The devil's in the details, and while I don't think anyone would get confused as to what the first column communicates, I could see potential for some readers to get distracted by that empty cell.

Of those two options @mmenestr supplied, I think "Use case" would work the best above the first column. It's short and reflects rhetoric we use regularly throughout our docs. 👍